src/components/editMeasure/details/MeasureDefinitions/MeasureDefinitions.tsx

```diff
@@ -244,14 +273,18 @@ const MeasureDefinitions = (props: MeasureDefinitionsProps) => {
                 <th scope="col" className="col-header">
                   Definition
                 </th>
+                <th scope="col" className="col-header">
+                  Action
+                </th>
               </tr>
             </thead>
             <tbody data-testId="measure-definitions-table-body">
               {visibleDefinitions?.length > 0 ? (
                 visibleDefinitions.map((measureDefinition, index) => (
-                  <MeasureMetaDataRow
```

Comment:
@mcmcphillips @sb-cecilialiu- To reuse this component, can we not pass the edit function instead?

Reply:
We could use MeasureMetaDataRow to take in a third variable for function and Add the view logic in a third column

Reply:
@adongare @mcmcphillips Changed, thanks!